lib/src/custom_items/app_drawer.dart

```diff
@@ -0,0 +1,123 @@
+                style: Theme.of(context).textTheme.headline6.copyWith(
+                      fontSize: 40,
+                    ),
+              ),
+            ),
+            decoration: BoxDecoration(
+              color: Theme.of(context).primaryColor,
+            ),
+          ),
+          ListTile(
+            leading: Text(
+              "Documentation",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Icon(Icons.open_in_new_rounded),
+            onTap: () => _launch("https://www.builtree.org/simulate/"),
+          ),
+          ListTile(
+            leading: Text(
+              "License",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Icon(Icons.open_in_new_rounded),
+            onTap: () => _launch(
+              "https://github.com/builtree/simulate/blob/master/LICENSE",
+            ),
+          ),
+          ListTile(
+            leading: Text(
+              "Github",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Icon(Icons.open_in_new_rounded),
+            onTap: () => _launch("https://github.com/builtree/simulate/"),
+          ),
+          ListTile(
+            leading: Text(
+              "Submit an issue",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Icon(Icons.open_in_new_rounded),
+            onTap: () => _launch(
+              "https://github.com/builtree/simulate/issues/new/choose",
+            ),
+          ),
+          ListTile(
+            leading: Text(
+              "App Version",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Text(
+              "1.0.0",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+          ),
+          ListTile(
+            leading: Text(
+              "Dark Mode",
+              style: Theme.of(context).textTheme.subtitle2.copyWith(
+                    fontSize: 20,
+                  ),
+            ),
+            trailing: Switch(
+              value: theme.darkTheme,
+              activeColor: Colors.black,
+              onChanged: (bool value) {
+                theme.toggleTheme();
+              },
+            ),
+          ),
+          InkWell(
+            child: SvgPicture.asset(
+              "assets/images/builtree.svg",
+              width: 100,
+            ),
+            onTap: () => _launch("https://www.builtree.org/"),
+          ),
+        ],
+      ),
```

Comment:
Let's make it similar to other tiles as:
Project By    [LOGO] redirecting to github.com/builtree